ArduCopter/ReleaseNotes.txt

```diff
@@ -1,5 +1,200 @@
+    e) EFI supports fuel pressure and ignition voltage reporting and battery failsafe
+    f) ICM45686 IMU support
+    g) ICM20602 uses fast reset instead of full reset on bad temperature sample (avoids occasional very high offset)
+    h) ICM45686 supports fast sampling
+    i) MAX31865 temp sensor support
+    j) MB85RS256TY-32k, PB85RS128C and PB85RS2MC FRAM support
+    k) MMC3416 compass orientation fix
+    l) MPPT battery monitor reliability improvements, enable/disable aux function and less spammy
+    m) Multiple USD-D1-CAN radar support
+    n) NMEA output rate configurable (see NMEA_RATE_MS) 
+    o) NMEA output supports PASHR message (see NMEA_MSG_EN)
+    p) OSD supports average resting cell voltage (see OSD_ACRVOLT_xxx params)
+    q) Rockblock satellite modem support
+    r) Serial baud support for 2Mbps (only some hardware supports this speed)
+    s) SF45b lidar filtering reduced (allows detecting smaller obstacles
+    t) SmartAudio 2.0 learns all VTX power levels)
+    u) UAVCAN ESCs report error count using ESC Telemetry
+    v) Unicore GPS (e.g. UM982) support
+    w) VectorNav 100 external AHRS support
+    x) 5 IMUs supported
+4) EKF related enhancements
+    a) Baro compensation using wind estimates works when climbing or descending (see BAROx_WCF_UP/DN)
+    b) External AHRS support for enabling only some sensors (e.g. IMU, Baro, Compass) see EAHRS_SENSORS
+    c) Magnetic field tables updated
+    d) Non-compass initial yaw alignment uses GPS course over GSF (mostly affects Planes and Rover)
+5) Control and navigation enhancements
+    a) AutoTune of attitude control yaw D gain (set AUTOTUNE_AXES=8)
+    b) Circle moode and Loiter Turns command supports counter-clockwise rotation (set CIRCLE_RATE to negative number)
+    c) DO_SET_ROI_NONE command turns off ROI
+    d) JUMP_TAG mission item support
+    e) Missions can be stored on SD card (see BRD_SD_MISSION)
+    f) NAV_SCRIPT_TIME command accepts floating point arguments
+    g) Pause/Resume returns success if mission is already paused or resumed
+    h) Payload Place enhancements
+        i) Descent speed is configurable (see PLDP_SPEED_DN)
+        ii) Manual release supported (detects pilot release of gripper)
+        iii) Post release delay is configurable (see PLDP_DELAY)
+        iv) Range finder range used to protect against premature release (see PLDP_RNG_MIN)
+        v) Touchdown detection threshold is configurable (see PLDP_THRESH)
+    i) Position controller angle max adjusted inflight with CH6 Tuning knob (set TUNE=59)
+    j) Surface tracking time constant allows tuning response (see SURFTRAK_TC)
+    k) Takeoff throttle max is configurable (see TKOFF_TH_MAX)
+    l) Throttle-Gain boost increases attitude control gains when throttle high (see ATC_THR_G_BOOST)
+    m) Waypoint navigation cornering acceleration is configurable (see WPNAV_ACCEL_C)
+    n) WeatherVane option to only use during takoffs and landings (see WVANE_ENABLE)
```

Comment:
```suggestion
    n) Added support for Weather Vaneing into the wing in AUTO and GUIDED (see WVANE_ENABLE)
```
I don't think the addition of the feature ever made it to the notes, maybe this is the first release.

Reply:
cool, txs added.